src/main/java/de/tum/in/www1/artemis/service/exam/StudentExamService.java

```diff
@@ -309,6 +315,22 @@ else if (exercise instanceof ModelingExercise) {
         return studentExams;
     }
 
+    /**
+     * Helper method to return a map for each user to their exercises
+     *
+     * @param studentExams the student exams of the users containing the exercises
+     * @return a map of the User as key, and a list of the users exercises as value
+     */
+    public Map<User, List<Exercise>> getExercisesOfUserMap(Set<StudentExam> studentExams) {
+        return studentExams.stream()
```

Comment:
The formatting looks really ugly. Can we put this on less lines?

Reply:
I totally agree here, unfortunately the style checks always fail if I modify this in any way which makes it less ugly.